packs/conditions/confused.json

```
@@ -47,7 +47,21 @@
                 "inMemoryOnly": true,
                 "key": "GrantItem",
                 "uuid": "Compendium.pf2e.conditionitems.Item.Off-Guard"
-            }
+            },
+            {
+                "key": "Note",
+                "selector": "damage-received",
+                "text": "PF2E.condition.confused.note",
+                "predicate": [
+                  {
+                    "or": [
+                      "item:type:spell",
+                      "action:strike"
```

I don't think this roll option is present in `damage-received`, but we probably want something broader anyway. Non-strike attacks should also count for this.
```suggestion
                      "origin:action:trait:attack"
```